src/main/scala/beam/analysis/cartraveltime/CarTripStatsFromPathTraversalEventHandler.scala

```diff
@@ -331,33 +332,6 @@ class CarTripStatsFromPathTraversalEventHandler(
     )
   }
 
-  /**
-    * Plots graph for average travel times per hour at iteration level
-    *
-    * @param dataset category dataset for graph genration
-    * @param iterationNumber iteration number
-    */
-  private def createIterationGraphForAverageCarTravelTime(dataset: CategoryDataset, iterationNumber: Int): Unit = {
```

Comment:
I'm using the github UI - was this unused and that was why it was removed?

Reply:
Yes, that's correct. Though if this method is going to be needed soon I can restore it and handle the static field somehow.